### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Wisdom132**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,102 +1,70 @@
-<h1 align="center" style="color:#2F3C7E">
-  Coinratecap.com
-</h1>
+# Getting Started with Create React App
```
Comment (Francaolisa) [inline README.md]: Shouldn’t we start with asking users to fork the project if they want to contribute? Also Avoid placing too many links,it causes distraction.

File: README.md [hunk 2]
```diff
@@ -1,102 +1,70 @@
...
-4. Commit your changes
-5. Push to the branch
-6. Open a pull request
+## Learn More
 
-For further help, you can check out the [FAQs](https://thenewboston.com/faq).
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
-Finally, feel free to shoot any queries or suggestions. Don't hesitate. ;)
+To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (Francaolisa) [inline README.md]: Can we have coding guide for contributors for instance : Instruct then in the need to comment on each code block - describing what they have done, maintain a particular coding pattern,  the branch to use etc.

File: README.md [hunk 3]
```diff
@@ -1,102 +1,70 @@
...
+This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
 
-## Getting Started
+### Analyzing the Bundle Size
 
-After cloning the repo, run the following command to install the project dependencies:
+This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
```
Comment (Francaolisa) [inline README.md]: Pls remove links and tell users what to do here in summary.

**Review by Francaolisa — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (justinecodez) [reply README.md]: I think for the coding guide except for the very commons ones we should have to discuss on the channel and see which ones are the most preferred by the majority personally I have not contributed to a lot of open source projects this is my first one.

File: README.md (hunk 1 above)
Comment (justinecodez) [reply README.md]: Noted that

File: README.md (hunk 2 above)
Comment (Francaolisa) [reply README.md]: Ok

**Review by Francaolisa — APPROVED** [review]

**Pull request merged**